### DISCUSSION TITLE
Model not learning, can't figure out where the problem is

### DISCUSSION THREAD

**Discussion by AmitShakhar** [mrdbourke/pytorch-deep-learning#898] [Q&A]
For some reason my model isn't able to predict correctly but ive gone over my code so many times and can't figure out what the problem is. Any help would be greatly appreciated.

https://colab.research.google.com/drive/1c_1iwc0ucOLC_Jw0h5ingLFDASB015Th?usp=sharing

**Comment by viti990** [comment]
it seems to me like it is learning, maybe let it learn for 5000 epochs
